lib/public/views/Runs/Overview/exportRunsTriggerAndModal.js

```diff
@@ -109,3 +109,19 @@ export const exportRunsModal = (runsOverviewModel, modalHandler) => {
         }),
     ]);
 };
+
+/**
+ * Builds a button which will open popover for data export
+ * @param {RunsOverviewModel} runsModel runs overview model
+ * @param {ModelModel} modalModel modal model
+ * @returns {Component} button
+ */
+export const exportRunsTriggerAndModal = (runsModel, modalModel) => h('button.btn.btn-primary.w-15.h2.mlauto#export-runs-trigger', {
+    disabled: runsModel.runs.match({
+        Success: (payload) => payload.length,
+        NotAsked: () => 0,
+        Failure: () => 0,
+        Loading: () => 0,
+    }) === 0,
```

Comment:
Now that I see it, I would rather return directly booleans instead of 0

Reply:
done